### PR TITLE
Added dynamic viewdistance thing per world + if elytra fly

### PR DESCRIPTION
**Pull request by charlie353535**
- Dynamic view distance per world
- Lowerable if someone is eflying

### CODE REVIEW

**Review by x4e — CHANGES_REQUESTED** [review]
Formatting

**Comment by charlie353535** [comment]
image

**Comment by x4e** [comment]
:-1:

**Comment by charlie353535** [comment]
fixed line endings

File: src/main/kotlin/dev/binclub/paperbin/transformers/TpsCompensation.kt [hunk 1]
```diff
@@ -110,7 +110,7 @@ object TpsCompensation: PaperBinFeature {
 					}
 					//PUTFIELD .e : I
 				}
-			}
+			}*/
```
Comment (x4e) [inline src/main/kotlin/dev/binclub/paperbin/transformers/TpsCompensation.kt]: Why are these changes relevant to this PR? Please make a new PR for these changes

Comment (charlie353535) [reply src/main/kotlin/dev/binclub/paperbin/transformers/TpsCompensation.kt]: oops

**Comment by x4e** [comment]
Thanks

**Comment by Tigermouthbear** [comment]
epic
